dotnet/dotnet-guestbook/src/backend/skaffold.yaml

```diff
@@ -0,0 +1,27 @@
+# To learn more about the skaffold.yaml schema visit
+# https://skaffold.dev/docs/references/yaml/
+apiVersion: skaffold/v2beta18
+kind: Config
```

Comment:
All of these dependent configurations are not modules yet until we name them, right?
```
kind: Config
metadata:
  name: backend
```

I think we need to add names to all of these `backend`/`frontnend` modules

Reply:
Fixed!